index.js

```diff
@@ -26,6 +26,8 @@ function Physics (mcData, world) {
   const lavaId = blocksByName.lava.id
   const ladderId = blocksByName.ladder.id
   const vineId = blocksByName.vine.id
+  const bubblecolumnId = blocksByName.bubble_column ? blocksByName.bubble_column.id : -1 // 1.13+
+  const airId = blocksByName.air.id
```

Comment:
Optional: air id is always 0 and it would be weird if it ever change, do we need to define it ?

Reply:
i thought the same thing, but i did it in the end to be consistent, ill place it with a 0 and add a comment saying its an air block.

Reply:
I might be too late to comment here but I would insist on using a const for air, as @IdanHo says it's more consistent, but it's also cleaner and easier to read. Using a literal here exposes implementation details we shouldn't have to see our care about. Replacing an accurately named const with a literal and a comment is counter productive.